cmd/estuary-shuttle/main.go

```diff
@@ -462,9 +473,25 @@ type User struct {
 
 	AuthToken       string `json:"-"` // this struct shouldnt ever be serialized, but just in case...
 	StorageDisabled bool
+	AuthExpiry      time.Time
 }
 
 func (d *Shuttle) checkTokenAuth(token string) (*User, error) {
+
+	val, ok := d.authCache.Get(token)
+	if ok {
+		usr, ok := val.(User)
```

Comment:
probably makes sense to put pointers to users in the cache, i think they get silently converted to pointers under the hood, and this might make things a little weird

Reply:
seems reasonable, done